MykSuite/src/main/java/com/infomaniak/lib/myksuite/ui/screens/MyKSuiteDashboardScreen.kt

```diff
@@ -0,0 +1,235 @@
+            expendedView = { LimitedFunctionalities(paddedModifier, dailySendingLimit) },
+        )
+        Spacer(Modifier.height(Margin.Medium))
+    }
+}
+
+@Composable
+private fun PaddedDivider(modifier: Modifier) {
+    Spacer(Modifier.height(Margin.Large))
+    HorizontalDivider(modifier)
+    Spacer(Modifier.height(Margin.Large))
+}
+
+// TODO: Add this when we'll have in-app payments
+@Composable
+private fun MyKSuitePlusPromotionCard(modifier: Modifier = Modifier, onButtonClicked: () -> Unit) {
+    Card(
+        modifier = modifier,
+        shape = RoundedCornerShape(Dimens.largeCornerRadius),
+        elevation = CardDefaults.elevatedCardElevation(defaultElevation = Dimens.cardElevation),
+        border = gradientBorder(),
+    ) {
+        Box(Modifier.padding(Margin.Medium)) {
+            Column(verticalArrangement = Arrangement.spacedBy(Margin.Medium)) {
+                Row(verticalAlignment = Alignment.CenterVertically) {
+                    Image(
+                        modifier = Modifier.width(88.dp),
+                        contentScale = ContentScale.FillWidth,
+                        imageVector = ImageVector.vectorResource(R.drawable.ic_logo_my_ksuite_plus),
+                        contentDescription = "My kSuite +",
+                    )
+                    WeightOneSpacer(minWidth = Margin.Large)
+                    Text(
+                        text = stringResource(R.string.myKSuiteDashboardFreeTrialTitle),
+                        style = MyKSuiteTheme.typography.labelMedium,
+                        color = MyKSuiteTheme.colors.primaryTextColor,
+                        modifier = Modifier
+                            .background(
+                                color = MyKSuiteTheme.colors.secondaryBackground,
+                                shape = RoundedCornerShape(Dimens.largeCornerRadius),
+                            )
+                            .padding(horizontal = Margin.Mini, vertical = Margin.Micro),
+                    )
+                }
+                Text(
+                    text = stringResource(R.string.myKSuiteDashboardFreeTrialDescription),
+                    style = MyKSuiteTheme.typography.bodySmallRegular,
+                    color = MyKSuiteTheme.colors.primaryTextColor,
+                )
+                Button(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .height(Dimens.buttonHeight),
+                    colors = ButtonDefaults.buttonColors(containerColor = MyKSuiteTheme.colors.driveButton),
+                    shape = RoundedCornerShape(Dimens.largeCornerRadius),
+                    onClick = onButtonClicked,
+                ) {
+                    Text(
+                        text = stringResource(R.string.myKSuiteDashboardFreeTrialButton),
+                        style = MyKSuiteTheme.typography.bodyMedium,
+                    )
+                }
```

Comment:
The text color of your button is wrong. You should probably reuse a `MyKSuitePrimaryButton()` component shared through the module or something to centralise all of this logic once and for all